modules/gapi/test/s11n/gapi_s11n_tests.cpp

```diff
@@ -460,6 +504,19 @@ TEST_F(S11N_Basic, Test_Bind_RunArgs_MatScalar) {
     }
 }
 
+TEST_F(S11N_Basic, Test_RunArg_RMat) {
```

Comment:
Need to add more tests. At least with several different runArgs (e.g. RMat, Mat, Scalar). Any other suggestions?

Reply:
Added one more test with several RunArgs